internal/controllers/signaturestats.go

```diff
@@ -0,0 +1,5 @@
+package controllers
+
+func GetStatsSignedData() string {
+	return "loadbalancer-request"
```

Comment:
What does this do?

Reply:
This is a constant that represents data being signed on stats request (as there is no payload), I just wrapped it as a function.

Reply:
@mpetrun5 do you think that it is better just to hold it as constant?

Reply:
This is a bit counter intuitive

Reply:
I refactored it to const